docs/rules/no-duplicate-imports.md

```diff
@@ -14,57 +16,47 @@ import { find } from 'module';
 
 This inspection reports any duplicated module in an import statement.
 
-The following patterns are considered problems:
+Example of **incorrect** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
 import { merge } from 'module';
-import path from 'another-module';
-import { find } from 'module';
-
-
-import { merge } from 'module';
+import something from 'another-module';
 import _, { find } from 'module';
 ```
 
-The following patterns are not considered problems:
+Example of **correct** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
-import { merge, find } from 'module';
-import path from 'another-module';
+import _, { merge, find } from 'module';
+import something from 'another-module';
 ```
 
 ## Options
 
 This rule takes one optional argument, an object with a single key, `includeExports` which is a `boolean`. It defaults to `false`.
 
-With this option set to `true`, the following patterns are considered problems:
+If re-exporting from an imported module, you should add the imports to the `import`-statement, and export that directly, not use `export ... from`.
+
+With this option set to `true`, the following examples is **incorrect** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: ["error", { includeExports: true }]*/
 
 import { merge } from 'module';
-import path from 'another-module';
 
 export { find } from 'module';
-
-
-import _ from 'module';
-const find = _.find;
-
-export { find as lodashFind } from 'module';
 ```
 
-The following patterns are not considered problems:
+Example of **correct** code for this rule:
```

Comment:
Changed this as well, as I was making the change you suggested for the top of the file 😄